core/common/src/main/java/alluxio/wire/AlluxioMasterInfo.java

```diff
@@ -0,0 +1,241 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.wire;
+
+import com.google.common.base.Objects;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Alluxio master descriptor.
+ */
+public class AlluxioMasterInfo {
+  private Capacity mCapacity;
+  private Map<String, String> mConfiguration;
+  private Map<String, Long> mMetrics;
```

Comment:
there is currently no mechanism to do that ... the web UI hardcodes the selection ... we could add the categorization to the metrics system, but I would prefer to add do that in a separate PR